src/3d/qgs3dmapsettings.h

```diff
@@ -431,6 +431,14 @@ class _3D_EXPORT Qgs3DMapSettings : public QObject, public QgsTemporalRangeObjec
     //! Returns the eye dome lighting distance value (contributes to the contrast of the image)
     int eyeDomeLightingDistance() const;
 
+    /**
+    * Sets whether to keep updating scene on zooming or keep the same amount of points visible
```

Comment:
this "stop updates" does not apply only to points, but pretty much all 3D entities - terrain, vector layers, tiled scene layers etc. So best to rephrase this to something like "Sets whether scene updates upon camera movement should be enabled", maybe even add a note that by default scene always gets updated and this option is good for debugging purposes.